postgres/datadog_checks/postgres/relationsmanager.py

```diff
@@ -35,7 +35,7 @@
   LEFT JOIN pg_namespace pn ON (pn.oid = pc.relnamespace)
  WHERE {relations}
    AND l.mode IS NOT NULL
-   AND pc.relname NOT LIKE 'pg_%%'
+   AND pc.relname NOT LIKE 'pg^_%%' ESCAPE '^'
```

Comment:
This looks fine but I'm curious as to why you set `^` as the escape character versus using the default `\`.

Reply:
Sorry, this was supposed to be a draft only for now - i have that changed back locally. I was kind of messing around with sql syntax options :)

Reply:
Hm though actually: flake8 throws [this error](https://www.flake8rules.com/rules/W605.html) when I use a standard escape. Additionally, postgresql has a warning in their [LIKE docs](https://www.postgresql.org/docs/current/functions-matching.html#FUNCTIONS-LIKE) that says: `If you have [standard_conforming_strings](https://www.postgresql.org/docs/current/runtime-config-compatible.html#GUC-STANDARD-CONFORMING-STRINGS) turned off, any backslashes you write in literal string constants will need to be doubled.`. I think I'll stick with the non-standard escape char for this clause.